test/new-e2e/pkg/utils/e2e/client/host.go

```diff
@@ -115,6 +116,7 @@ func (h *Host) Execute(command string, options ...ExecuteOption) (string, error)
 }
 
 func (h *Host) executeAndReconnectOnError(command string) (string, error) {
+	h.context.T().Logf("Executing command: `%s`", command)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should we try parse for security keys? Not sure if we ever do it, we might send commands prefixed by env variables